src/components/Accordion/Accordion.jsx

```diff
@@ -1,29 +1,74 @@
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'clsx';
 import {AccordionContext} from './Accordion.context';
 import {AccordionItem} from './AccordionItem';
 import styles from './Accordion.scss';
 
-export const Accordion = ({children, openByDefault, isReversed, className, ...props}) => {
-    const [currentItem, setCurrentItem] = useState(openByDefault ? openByDefault : null);
-    const [reversed] = useState(isReversed);
+export const Accordion = ({children, defaultOpenedItemId, openedItemId, isReversed, className, isMultipleOpenable, ...props}) => {
+    const [stateOpenedItemId, setStateOpenedItemId] = useState(defaultOpenedItemId);
```

Comment:
If the component is controlled, it should not have a state. Only way to make this clean and stable with state hooks is to have separate components for controlled and uncontrolled versions